### DISCUSSION TITLE
I hope the Copilot won't be available solely on the subscription basis

### DISCUSSION THREAD

**Discussion by Lippiece** [community/community#17917] [Copilot Conversations]
Basically, today I received the invite, tried it and now I'm scared because it's so good and I'm getting used to it, but it [will be commercially](https://copilot.github.com/#faq-will-there-be-a-paid-version) available in the future. I haven't a slightest doubt the technical preview will be successful, but if it will be available with a subscription, as popular today, I don't think I could handle it financially and emotionally.

**Comment by PurHur** [comment]
I mean i would pay some bucks privately private but I think this tool will be more expensive. Look for an employer that pays for this great tool.
